src/components/ha-selector/ha-selector-select.ts

```diff
@@ -209,6 +209,14 @@ export class HaSelectSelector extends LitElement {
           `
         )}
       </ha-select>
+      ${!this.required && this.value
```

Comment:
```suggestion
      ${!this.required && !this.disabled && this.value
```
Should we hide the clear icon if the field is disabled?

Reply:
We should at least disable the button 👍🏼

Reply:
I think removing the button is better. It's like there is no value